starkbank/merchantsession/__merchantsession.py

```diff
@@ -0,0 +1,82 @@
+from starkcore.utils.resource import Resource
+from ..utils import rest
+from starkcore.utils.api import from_api_json
+from .allowedinstallment.__allowedinstallment import AllowedInstallment
+from .allowedinstallment.__allowedinstallment import _sub_resource as _allowed_installments_sub_resource
+from starkcore.utils.checks import check_date, check_datetime
+from .__purchase import _resource as purchase_resource
+
+
+class MerchantSession(Resource):
+    """# MerchantSession object
+    Check out our API Documentation at https://starkbank.com/docs/api#merchant-session
+    """
+
+    def __init__(self, allowed_funding_types, allowed_installments, expiration, id=None, allowed_ips=None, challenge_mode=None, created=None, status=None, tags=None, updated=None, uuid=None):
+        Resource.__init__(self, id=id)
+
+        self.allowed_funding_types = allowed_funding_types
+        self.allowed_installments = _parse_allowed_installments(allowed_installments)
+        self.allowed_ips = allowed_ips
+        self.challenge_mode = challenge_mode
+        self.expiration = expiration
+        self.status = status
+        self.tags = tags
+        self.created = check_datetime(created)
+        self.updated = check_datetime(updated)
+        self.uuid = uuid
+
+
+_resource = {"class": MerchantSession, "name": "MerchantSession"}
+
+
+def _parse_allowed_installments(allowed_installments):
+    if allowed_installments is None:
+        return None
```

Comment:
Não é melhor deixar o retorno sempre sendo uma lista? Pra não quebrar a assinatura do método.
```suggestion
        return []
```